src/main/java/com/eternalcode/lobbyheads/head/particle/ParticleController.java

```diff
@@ -0,0 +1,41 @@
+package com.eternalcode.lobbyheads.head.particle;
+
+import com.eternalcode.lobbyheads.configuration.implementation.HeadsConfiguration;
+import com.eternalcode.lobbyheads.head.event.HeadUpdateEvent;
+import com.eternalcode.lobbyheads.position.Position;
+import com.eternalcode.lobbyheads.position.PositionAdapter;
+import org.bukkit.Location;
+import org.bukkit.Server;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+import java.util.UUID;
+
+public class ParticleController implements Listener {
```

Comment:
Nie jestem pewien ale chyba lepiej zrobić ten cały kodzik po prostu w controllerze od interact'a

Reply:
I prefer to have cosmetic features implemented in separate classes.